src/main/java/step1/Calculator.java

```diff
@@ -0,0 +1,28 @@
+package step1;
+
+public class Calculator {
+	private int result;
+	private final Separator separator;
+
+	public Calculator(String input) {
+		Separator separator = new Separator(input);
+		result = Validator.validateNumber(separator.getTerm());
+		this.separator = separator;
+	}
+
+	public int start() {
+		for (int i = 1; i <= separator.getFormula().size() - 2; i += 2) {
+			calculate(Validator.validateSign(separator.getSignTerm(i)),
+				Validator.validateNumber(separator.getTerm(i + 1)));
+		}
```

Comment:
```suggestion
            String sign = Validator.validateSign(separator.getSignTerm(i));
            int number = Validator.validateNumber(separator.getTerm(i + 1));
            calculate(sign, number);
```
파라미터로 변수를 사용하거나 메서드 길이가 너무 길어진다면 한번 끊고 가는게 조금더 읽기 쉬울 것 같아요!